docs/cimpl6ToolingReference.md

```diff
@@ -498,16 +497,34 @@ For many users, especially those with experience in object-oriented modeling, th
 
 ### Data Dictionary Export
 
-The Data Dictionary is a Microsoft Excel file containing a simplified, flattened list of model `Elements` and value sets, extracted from the FHIR profiles. The spreadsheet has five tabs:
+The Data Dictionary is a Microsoft csv file containing a simplified, flattened list of model `Elements` and value sets, extracted from the FHIR profiles. The spreadsheet has five tabs:
```

Comment:
The file type is still Microsoft Excel (.xlsx), and there are now only four sheets in the file due to the removal of the _Key_ sheet.